vsphere/requirements.in

```diff
@@ -1 +1,2 @@
-vsphere-automation-sdk==1.0.0
+# When vmware publishes new release with this fixes we'll build it and publish it to artifactory.stackstate.io
```

Comment:
Maybe add this comment as a `# TODO`, then the IDE recognizes and reminds us about it.